### PR TITLE
welcomePage+App

### PR DESCRIPTION
**Pull request by VolodymyrBab**

### CODE REVIEW

File: src/components/WelcomePage/index.jsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import iconImage from '../../images/welcome-icon.png';
```
Comment (SergeyMikhailuk) [inline src/components/WelcomePage/index.jsx:2]: import iconImage from 'images/welcome-icon.png';

File: src/components/WelcomePage/index.jsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+import sprite from '../../images/sprite.svg';
```
Comment (SergeyMikhailuk) [inline src/components/WelcomePage/index.jsx:4]: import sprite from 'images/sprite.svg';

File: src/components/WelcomePage/styled.js
Comment (SergeyMikhailuk) [inline src/components/WelcomePage/styled.js:1]: good

File: src/App.jsx
[the remark this answers is not in the document]
Comment (VolodymyrBab) [reply src/App.jsx:1]: ? budem bez App?

**Comment by VolodymyrBab** [comment]
mozna

**Review by SergeyMikhailuk — APPROVED** [review]

**Pull request merged**